auth/src/main/java/com/firebase/ui/auth/util/OldCredentialsApiHelper.java

```diff
@@ -0,0 +1,101 @@
+package com.firebase.ui.auth.util;
+
+import android.app.Activity;
+import android.support.annotation.NonNull;
+
+import com.google.android.gms.auth.api.Auth;
+import com.google.android.gms.auth.api.credentials.Credential;
+import com.google.android.gms.common.api.GoogleApiClient;
+import com.google.android.gms.common.api.Result;
+import com.google.android.gms.common.api.ResultCallback;
+import com.google.android.gms.common.api.Status;
+import com.google.android.gms.tasks.Continuation;
+import com.google.android.gms.tasks.Task;
+import com.google.android.gms.tasks.TaskCompletionSource;
+
+/**
+ * A {@link com.google.android.gms.tasks.Task Task} based wrapper for the Smart Lock for Passwords
+ * API.
+ */
+@Deprecated
+public class OldCredentialsApiHelper {
```

Comment:
Let's pick a better name for this, maybe `ActivityCredentialsApiHelper` or something.  "Old" as a prefix does not help to show what was wrong with it.

Also I'd argue that we should keep this class named `CredentialsApiHelper` and the new class is the one that should have a new name since it reduces churn.